www/routes/docs/[...slug].tsx

```diff
@@ -50,7 +50,10 @@ export default createMainPage(function DocsPage(props: PageProps<Data>) {
   if (props.data.page.data.description) {
     description = String(props.data.page.data.description);
   }
-  let installer = props.data.base !== '/' ? `${props.data.base}/pls` : `${props.url.protocol}//${props.url.host}/pls`
+  let base = props.data.base?.trim().replace(/\/$/, "") ?? "";
```

Comment:
What about this?
```suggestion
  let base = props.data.base ? new URL(props.data.base).origin : "";
```

Reply:
The base url could have a path component to it:

```ts
new URL('https://frontside.com/platformscript').origin
// https://frontside.com
```